### ISSUE TITLE
Some issues with Mac

### ISSUE THREAD

**Issue by idr4n** [ahrm/sioyek#115]
Hi, and thanks for this very cool project. I love the idea of having flexible PDF readers that are keyboard-centered. I have been using Zathura in Mac for some time now, but Sioyek has really nice features.

I have encountered two issues so far using the Mac build provided on the release page (I'm running MacOS v12.1, M1 silicon).

- First, the application stops responding completely when pressing `t` to access the table of contents. I have to force quit and restart the application.
- Second, if I try to open a pdf file from the file explore (finder in Mac) using "open with" and selecting Sioyek, the application launches but does not open the selected file (is there a command to open pdf files from the terminal?). It seems like I can only open files from within Sioyek by pressing `o`.

Everything else seems to be working fine.

Thanks in advance!

**Comment by ahrm** [comment]
Hi! Thanks for you feedback.
Currently I don't have access to a Mac computer but I will check out these issues as soon as I have access to one.

**Comment by ivanddm** [comment]
Thanks again. Sure I understand!

> By the way you can open documents from command line using
>`sioyek file_name.pdf`

Actually, I tried that but the command is not available in either zsh or bash. I will try to grab it from the application directory if I find it and add it to my path.

**Comment by sbnietert** [comment]
I am having the same issues. This command works if you add the executable (within the app package) to your path.

**Comment by sbnietert** [comment]
I have a hacky work-around for complaint (ii). I'm sure there is a standard way to resolve this but know nothing about macOS development. Instead, I used Automator to create a new application which runs the shell script
"/Applications/sioyek.app/Contents/MacOS/sioyek $1".

**Comment by ma-con** [comment]
> I have a hacky work-around for complaint (ii). I'm sure there is a standard way to resolve this but know nothing about macOS development. Instead, I used Automator to create a new application which runs the shell script
> "/Applications/sioyek.app/Contents/MacOS/sioyek $1". (edit: make sure to set "Pass input" to "as arguments")

I just had the same problem and found this issue, thanks for the idea with the automator wrapper. 

I’ve slightly changed the shell script to use the “open” command, as otherwise I was unable two open more than one file at the same time, the second file would only open once sioyek with the first file was closed. Using  `open -a` does not have this problem. 

Btw: While googling for a solution, I stumbled upon this  https://doc.qt.io/qt-5/qfileopenevent.html#macos-example  Changing the plist is easy but I unfortunately have no idea how to implement the other necessary part.
